src/gridcoinresearchd.cpp

```diff
@@ -48,10 +49,13 @@ bool AppInit(int argc, char* argv[])
         //
         // Parameters
         //
-        // If Qt is used, parameters/bitcoin.conf are parsed in qt/bitcoin.cpp's main()
-        ParseParameters(argc, argv);
-
-        if (mapArgs.count("-?") || mapArgs.count("-help"))
+        // If Qt is used, parameters/gridcoinresearch.conf are parsed in qt/bitcoin.cpp's main()
+        std::string error;
+        if (!gArgs.ParseParameters(argc, argv, error)) {
+            tfm::format(std::cerr, "Error parsing command line arguments: %s\n", error);
+            return EXIT_FAILURE;
+        }
+        if (argc < 2 || HelpRequested(gArgs) || gArgs.IsArgSet("-version"))
```

Comment:
```suggestion
        if (argc < 2 || HelpRequested(gArgs))
```

Looks like `-version` is handled explicitly below (this caused `-version` to print the help message instead).